docs/contributing/upstream-issues.md

```diff
@@ -0,0 +1,76 @@
+# Upstream Issues
```

Comment:
I don't feel this is the right place for this page. It fits the wiki better because it can change frequently

Reply:
docs can also change frequently if you let them change frequently :) just use the inbuilt github edit button and update them. better have stuff in one place than in different ones

Reply:
ok you convinced me